bundle/src/main/java/com/adobe/acs/commons/fam/impl/ActionManagerImpl.java

```diff
@@ -278,9 +278,10 @@ private void deferredWithResolver(
                 if (!closesResolver) {
                     logCompletetion();
                 }
-            } catch (Exception ex) {
+            } catch (Throwable t) {
```

Comment:
So here is where I think Throwable is wrong since you aren't rethrowing the exception. I think `catch (Exception|RuntimeException) ex) ` would be a better choice.